sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 2.0.0-beta.2 (Unreleased)
 
+- Added `clientId` to the `AuthenticationRecord` type.
+- `AuthenticationRecord` no longer has a `serialize` method. This method is now called `serializeAuthenticationRecord` and is a function exported at the top level of the Identity library, similar to `deserializeAuthenticationRecord`.
+- `serializeAuthenticationRecord` now serializes into a JSON string with camel case properties. This makes it re-usable across languages.
```

Comment:
Is this a breaking change?

Reply:
From 2.0.0-beta.1 only

Reply:
Should we add [breaking] and add when it was added and where its changing?

Reply:
Not that I remember. Only from the last GA.